apps/src/applab/Exporter.js

```diff
@@ -455,7 +462,7 @@ export default {
       applabLocalePath: "https://studio.code.org/blockly/js/en_us/applab_locale.js",
       appOptionsPath: "appOptions.j",
       fontPath: fontAwesomeWOFFPath,
-      applabApiPath: "https://studio.code.org/blockly/js/applab-api.js",
+      applabApiPath: "https://studio.code.org/blockly/js/applab-api.min.js",
```

Comment:
Should this vary based on whether we were able to grab the minified version?

Reply:
In this case, we are simply linking to the script from our live production site (no offline requirement, so we don’t download the file)